FoxHound.Client/src/Posts/PostContentEditor.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import MarkdownEditor from "react-simplemde-editor";
+import hljs from "highlight.js";
+
+import "easymde/dist/easymde.min.css";
+import "highlight.js/styles/solarized-light.css";
+
+interface IProps {
+  id: string;
+  content: string;
+  setContent: React.Dispatch<React.SetStateAction<string>>;
+}
+
+const PostContentEditor: React.FC<IProps> = (props) => {
+  const { id, content, setContent } = props;
+
+  return (
+    <div>
+      <MarkdownEditor
+        id={id}
+        options={{
+          hideIcons: ["guide"],
+          showIcons: ["code", "strikethrough", "table", "horizontal-rule"],
+          renderingConfig: { codeSyntaxHighlighting: true, hljs: hljs },
+          autosave: {
+            enabled: true,
+            uniqueId: id,
```

Comment:
This saves the content to local storage in case the page is refreshed, etc. This gets cleared automatically after form submit.